### ISSUE TITLE
Misc security related browser console errors

### ISSUE THREAD

**Issue by MichalTuleja** [LiskHQ/lisk-explorer#453]
They do not break any functional part of code, but there is a significant number of them.

```
Refused to load the font '<URL>' because it violates the following Content Security Policy directive: "font-src 'self' <URL>".
```

```
Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAGVU[...]XAFAmKZU=' because it violates the following Content Security Policy directive: "font-src 'self' https://fonts.gstatic.com".
```

This problem was noticed under Chrome Browser Version 65.0.3325.181 (Official Build) (64-bit) on MacOS High Sierra (10.13.3).

**Comment by MichalTuleja** [comment]
This screen comes from Explorer 1.4.3 on MacOS X 10.13.3, Chrome Version 64.0.3282.186 (Official Build) (64-bit)
![screen shot 2018-03-21 at 17 39 36](https://user-images.githubusercontent.com/4547510/37723561-f2e73dfc-2d2e-11e8-8af5-722801555fd1.png)

**Comment by MichalTuleja** [comment]
The problem is fixed in the `2.2.0`.
